docs/observability-deployment.md

```diff
@@ -1 +1,152 @@
 
+
+# Observability with Open Liberty
+
+
+The following document covers various topics for configuring and integrating your Open Liberty runtime with monitoring tools in the OpenShift cluster.
+
+## How to deploy Kibana dashboards to monitor Open Liberty logging events  
+
+Kibana dashboards are provided for visualizing events from the Open Liberty runtime.
+
+To leverage these dashboards the logging events must be emitted in JSON format to standard-out. For information regarding how to configure an Open Liberty image with JSON logging please see [the following](https://github.com/OpenLiberty/ci.docker#logging).
+
+Retrieve available Kibana dashboards tuned for Open Liberty logging events [here](https://github.com/OpenLiberty/open-liberty-operator/tree/master/deploy/dashboards/logging).
+
+For information regarding how to import Kibana dashboards see the official documentation [here](https://www.elastic.co/guide/en/kibana/5.6/loading-a-saved-dashboard.html).
+
+For effective management of logs emitted from applications to deploy your own Elasticsearch, FluentD and Kibana (EFK) stack. For more information see the following [guide](https://kabanero.io/guides/app-logging/). 
```

Comment:
For effective management of logs emitted from applications, deploy your own Elasticsearch, FluentD and Kibana (EFK) stack. For more information see this guide.